lib/node_modules/@stdlib/blas/base/ssyr/README.md

```diff
@@ -183,21 +183,33 @@ console.log( A );
 ### Usage
 
 ```c
-TODO
+#include "stdlib/blas/base/ssyr.h"
 ```
 
-#### TODO
+#### c_ssyr( order, uplo, N, alpha, \*X, strideX, \*A, LDA )
 
-TODO.
+Performs the symmetric rank 1 operation `A = α*x*x^T + A`
 
 ```c
-TODO
+float A[] = { 1.0f, 0.0f, 0.0f, 2.0f, 1.0f, 0.0f, 3.0f, 2.0f, 1.0f };
+float x[] = { 1.0f, 2.0f, 3.0f };
+
+c_ssyr( 102, 121, 3, 1.0f, x, 1, A, 3 );
```

Comment:
Do not hardcode enum values. Use enum identifiers. The values are subject to change. Enums should be treated as opaque constants.